sycl/include/CL/sycl/INTEL/esimd/esimd_memory.hpp

```diff
@@ -158,14 +158,15 @@ scatter(T *p, simd<T, n * ElemsPerAddr> vals, simd<uint32_t, n> offsets,
                             pred.data());
 }
 
-// TODO @rolandschulz
-// Should follow existing std::simd naming for similar APIs - "copy_from" and
-// "copy_to" to avoid confusion.
-//
 /// Flat-address block-load.
 /// \ingroup sycl_esimd
+// TODO normally, this function should just delegate to
+// simd::copy_from for the deprecation period, but separate implementations are
+// needed for now, as simd::copy_from does not support cache hints yet.
+// This API, even though deprecated, can't be removed until then.
 template <typename T, int n, CacheHint L1H = CacheHint::None,
           CacheHint L3H = CacheHint::None>
+__SYCL_DEPRECATED("Replaced by simd::copy_from")
```

Comment:
NIT. Consider rephrasing to "XYZ is deprecated. Use XZZ instead." to align with other deprecation warnings.

Reply:
ok